src/main/resources/swagger/api-docs.yaml

```diff
@@ -648,58 +647,57 @@ paths:
-      # parameters:
-      #   - name: namespace
-      #     in: query
-      #     description: Namespace of method.
-      #     required: false
-      #     type: string
-      #   - name: name
-      #     in: query
-      #     description: Name of method.
-      #     required: false
-      #     type: string
-      #   - name: snapshotId
-      #     in: query
-      #     description: Snapshot ID of method.
-      #     required: false
-      #     type: number
-      #   - name: synopsis
-      #     in: query
-      #     description: Synopsis of method.
-      #     required: false
-      #     type: string
-      #   - name: documentation
-      #     in: query
-      #     description: Documentation of method.
-      #     required: false
-      #     type: string
-      #   - name: owner
-      #     in: query
-      #     description: Owner of method.
-      #     required: false
-      #     type: string
-      #   - name: payload
-      #     in: query
-      #     description: Payload of method in WDL.
-      #     required: false
-      #     type: string
-      #   - name: entityType
-      #     in: query
-      #     description: Type of Method - Task or Workflow
-      #     required: false
-      #     type: string
-      #   - name: excludedFields
-      #     in: query
-      #     description: Fields to exclude in JSON response.
-      #     required: false
-      #     type: string
-      #   - name: includedFields
-      #     in: query
-      #     description: Fields to include in JSON response
-      #     required: false
-      #     type: string
+      parameters:
+        - name: namespace
+          in: query
+          description: Namespace of method.
+          required: false
+          type: string
+        - name: name
+          in: query
+          description: Name of method.
+          required: false
+          type: string
+        - name: snapshotId
+          in: query
+          description: Snapshot ID of method.
+          required: false
+          type: number
+        - name: synopsis
+          in: query
+          description: Synopsis of method.
+          required: false
+          type: string
+        - name: documentation
+          in: query
+          description: Documentation of method.
+          required: false
+          type: string
+        - name: owner
+          in: query
+          description: Owner of method.
+          required: false
+          type: string
+        - name: payload
+          in: query
+          description: Payload of method in WDL.
+          required: false
+          type: string
+        - name: entityType
+          in: query
+          description: Type of Method - Task or Workflow
+          required: false
+          type: string
+        - name: excludedFields
+          in: query
+          description: Fields to exclude in JSON response.
+          required: false
+          type: string
+        - name: includedFields
```

Comment:
I also can't get this to work. Tried using `payload` which is returned when you get the individual method or config, but no dice. The query looks correctly formed so this is either the param orchestration is sending or a bug in agora that doesn't honor it.